### PR TITLE
qe: add doc on required postgres package for 2.1.0

### PR DESCRIPTION
**Pull request by kbrwn**
Document to help users that may not have postgresql-contrib package installed. This package is required when installing QE 2.1.0 and above.

### CODE REVIEW

File: quay-enterprise/installing-contributor-extensions-postgres.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Installing Contributor Extensions for Postgres
+
+As of release 2.1.0 Quay Enterprise requires the [pg_trgm](https://www.postgresql.org/docs/current/static/pgtrgm.html) extension when using Postgres. This extension ships as part of the [Additional Supplied Modules](https://www.postgresql.org/docs/current/static/contrib.html). The requirements to allow extensions to be installed can be satisfied via package management tools on Debian and RedHat-based systems.
```
Comment (joshix) [inline quay-enterprise/installing-contributor-extensions-postgres.md]: s/2.1.0/2.1.0,/
s/Postgres/PostgreSQL/
s/ships as/is/
replace last sentence: "These modules can be installed with standard package management tools on Debian and RedHat-based systems."

Comment (joshix) [inline quay-enterprise/installing-contributor-extensions-postgres.md]:
```
Please [use reference-style links][style-reflinks] in new documents.

[style-reflinks]: https://github.com/coreos/docs/blob/master/STYLE.md#marking-down-the-link

File: quay-enterprise/installing-contributor-extensions-postgres.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Debian/Ubuntu
+
+```
+# apt-get update
+# apt-get install postgresql-contrib -y
```
Comment (joshix) [inline quay-enterprise/installing-contributor-extensions-postgres.md]: s/-y// (let them get the prompt)

File: quay-enterprise/installing-contributor-extensions-postgres.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# yum install postgresql-contrib
+```
+## Fedora
+
+```
+# dnf install postgresql-contrib
+```
+
+## Docker 
+The [library/postgres](https://hub.docker.com/_/postgres/) container includes `postgresql-contrib` package by default.
```
Comment (joshix) [inline quay-enterprise/installing-contributor-extensions-postgres.md]: s/includes/includes the/
s/by default//

File: quay-enterprise/installing-contributor-extensions-postgres.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+If using a container based on an OS above simply modify the Dockerfile to install `postgresql-contrib` package. If using an OS base that is not listed in may be required install `contrib` from [source](https://www.postgresql.org/docs/current/static/contrib.html).
```
Comment (joshix) [inline quay-enterprise/installing-contributor-extensions-postgres.md]: "If using a container based on one of the above operating systems, modify the container's Dockerfile to install the `postgresql-contrib` package. Containers built `FROM` other operating systems may need to [install the additional modules from source][modules-src-install]."

**Comment by joshix** [comment]
@kbrwn Would `postgres-additional-modules.md` be a better file name?

@josephschorr If so renamed, this file will end up at https://coreos.com/quay-enterprise/docs/latest/postgres-additional-modules.html` after edits, merge, release, and docs sync.

**Comment by jzelinskie** [comment]
Add a section about RDS having it on by default as well

**Pull request merged**